### PR TITLE
Sample jupiter notebook

### PR DESCRIPTION
**Pull request by navalev**
sample jupiter notebook for analyzer and anonymizer

### CODE REVIEW

**Comment by navalev** [comment]
@omri374 is it betther to save as a py or ipynb file?

**Comment by omri374** [comment]
@navalev looks great. One thing I would consider is to create markdown cells instead of comments. See example here: https://github.com/marcotcr/checklist/blob/master/notebooks/tutorials/1.%20Generating%20data.ipynb

It's better to save as ipynb as GH renders it as a notebook this way. 

The anonymizer interface requires some polishing for a Python experience IMHO but that's not for this user story.

File: docs/samples/python/presidio_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    "print(anonymized_results)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```
Comment (SharonHart) [inline docs/samples/python/presidio_notebook.ipynb:147]: empty cell

File: docs/samples/python/presidio_notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,149 @@
+{
```
Comment (SharonHart) [inline docs/samples/python/presidio_notebook.ipynb:1]: I've seen notebooks starting with pip install, do we want to add it?

**Review by TessFerrandez — APPROVED** [review]
Agree with Omri about the comments, but sample is very helpful

**Comment by navalev** [comment]
@omri374 I've extracted the comments to markdown cells and added more documentation. please review

File: docs/samples/python/presidio_notebook.ipynb (hunk 2 above)
Comment (navalev) [reply docs/samples/python/presidio_notebook.ipynb:1]: I'm assuming the analyzer and anonymizer are already installed when running this.

File: docs/samples/python/presidio_notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    "from presidio_anonymizer import AnonymizerEngine\n",
+    "from presidio_anonymizer.entities import AnonymizerRequest"
+   ]
+  },
+  {
+   "source": [
+    "# Analyze Text for PII Entities\n",
+    "\n",
+    "<br>Using Presidio Analyzer, analyze a text to identify PII entities. \n",
+    "<br>The Presidio analyzer is using pre-defined entity recignizers, and offers the otion to create custom recognizers.\n",
```
Comment (omri374) [inline docs/samples/python/presidio_notebook.ipynb:41]:
```suggestion
    "<br>The Presidio analyzer is using pre-defined entity recognizers, and offers the option to create custom recognizers.\n",
```

File: docs/samples/python/presidio_notebook.ipynb [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+   "source": [
+    "analyzer = AnalyzerEngine()\n",
+    "analyzer_results = analyzer.analyze(text=text_to_anonymize, entities=[\"PHONE_NUMBER\"], language='en')\n",
+    "\n",
+    "print(analyzer_results)"
+   ]
+  },
+  {
+   "source": [
+    "# Create Custom PII Entity Regocnizers\n",
```
Comment (omri374) [inline docs/samples/python/presidio_notebook.ipynb:75]:
```suggestion
    "# Create Custom PII Entity Recognizers\n",
```

File: docs/samples/python/presidio_notebook.ipynb [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+    "\n",
+    "<br>Presidio Analyzer comes with a pre-defined set of entity recognizers. It also allows adding new recognizers without changing the analyzer base code,\n",
+    "<b>by creating custom recognizers. \n",
+    "<br>In the following example, we will create two new recognizers of type PatternRecognizer to identify titles and pronouns in the analyzed text.\n",
```
Comment (omri374) [inline docs/samples/python/presidio_notebook.ipynb:79]: nit...
```suggestion
    "<br>In the following example, we will create two new recognizers of type `PatternRecognizer` to identify titles and pronouns in the analyzed text.\n",
```

File: docs/samples/python/presidio_notebook.ipynb [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+    "<br>A PatternRecognizer is PII entity recognizer using regular expressions or deny-lists.\n",
```
Comment (omri374) [inline docs/samples/python/presidio_notebook.ipynb:80]:
```suggestion
    "<br>A `PatternRecognizer` is a PII entity recognizer which uses regular expressions or deny-lists.\n",
```

File: docs/samples/python/presidio_notebook.ipynb [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+    "\n",
+    "<br>The following code sample will:\n",
+    "<ol>\n",
+    "<li>Create custom recognizers</li>\n",
+    "<li>Add the new custom recognizers to the analyzer and call the analyzer </li>\n",
```
Comment (omri374) [inline docs/samples/python/presidio_notebook.ipynb:85]:
```suggestion
    "<li>Add the new custom recognizers to the analyzer</li>\n",
```

File: docs/samples/python/presidio_notebook.ipynb [hunk 8]
```diff
@@ -0,0 +1,193 @@
...
+    "<li>Call analyzer to get results for the new recognizers</li>\n",
```
Comment (omri374) [inline docs/samples/python/presidio_notebook.ipynb:86]:
```suggestion
    "<li>Call analyzer to get results from the new recognizers</li>\n",
```

File: docs/samples/python/presidio_notebook.ipynb [hunk 9]
```diff
@@ -0,0 +1,193 @@
...
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "converted_analyzer_results = [{\"start\": analyzer_results[i].start, \"end\": analyzer_results[i].end, \"score\": analyzer_results[i].score, \"entity_type\": analyzer_results[i].entity_type}          for i in range(len(analyzer_results))]\n",
```
Comment (omri374) [inline docs/samples/python/presidio_notebook.ipynb:155]: We have a built in mechanism for this:
```suggestion
    "converted_analyzer_results = [result.to_dict() for result in analyzer_results]\n",
```

**Review by omri374 — APPROVED** [review]

**Pull request merged**